### PR TITLE
Fix action numbers (/handle UI default theme update)

### PR DESCRIPTION
**Pull request by Skyedra**
## About the PR
Companion PR to this [engine change](https://github.com/space-wizards/RobustToolbox/pull/3973) to handle UI default theme updates.  Please see that page for full details.

Requires https://github.com/space-wizards/RobustToolbox/pull/3973
Requires https://github.com/space-wizards/RobustToolbox/pull/4042

**Media**
Before:

Screenshot_20230428_212324 before change

After:

Screenshot_20230428_214420 after 9

(Note the white numbers are back)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: Returned the action hotkey numbers to their UI holding cells after mime let them out.

### CODE REVIEW

**Comment by AJCM-git** [comment]
linter fail

**Comment by Skyedra** [comment]
> linter fail

Yeah, I'm guessing linter doesn't currently understand theme paths

File: Content.Client/UserInterface/Systems/Actions/Controls/ActionPageButtons.xaml [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <Control HorizontalExpand="True" SizeFlagsStretchRatio="2"/>
         <Label Text="1" SizeFlagsStretchRatio="1" Name="Label" Access="Public" />
         <Control HorizontalExpand="True" SizeFlagsStretchRatio="2"/>
-        <TextureButton TexturePath="/Textures/Interface/Nano/right_arrow.svg.192dpi.png"
```
Comment (ElectroJr) [inline Content.Client/UserInterface/Systems/Actions/Controls/ActionPageButtons.xaml:16]: The icons should probably just be moved to the default theme instead using a relative path like this. I.e., just move them into  `Interface\Default`.

File: Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml [hunk 2]
```diff
@@ -7,7 +7,9 @@
     Orientation="Horizontal"
     HorizontalExpand="False"
 >
-    <BoxContainer Orientation="Vertical">
+    <BoxContainer
+        Orientation="Vertical"
+        Margin="0 0 0 -9"> <!-- Margin is a big hack since for some mysterious reason, the container is getting 9px added to it.... somewhere. -->
```
Comment (ElectroJr) [inline Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml:12]: I think this is just an issue with out `DefaultGameScreen` lays out controls when there is insufficient space. So instead of just fudging a margin here the layout/measure logic should just be fixed in a separate PR.

**Review by ElectroJr — DISMISSED** [review]
.

File: Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml (hunk 2 above)
Comment (Skyedra) [reply Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml:12]: It's been a while so I don't quite remember, but I recall this being pretty unappetizing at default res when I tested it -- either because the side bar clips into top left menu or because the arrows clip off the bottom of the screen (I don't remember which, though).  So it's going to look a bit jank at default resolution until someone gets around to fixing it properly.  

IMO it would be better to cheat it temporarily so as to not force the playerbase to have ugly ui issues as it may be some time until properly fix (unless you've heard someone is looking into this, of course).

Comment (ElectroJr) [reply Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml:12]: I generally wouldn't mind a temporary fix, but this doesn't even fix it, it still breaks when using a different UI scale or resolution, so I'd rather just not add in temporary hacks that might cause confusion later on when trying to fix the actual issue.


https://github.com/space-wizards/space-station-14/assets/60421075/0dfa6ec6-3dbb-4616-9c42-f49168f69566

Comment (Skyedra) [reply Content.Client/UserInterface/Systems/Actions/Widgets/ActionsBar.xaml:12]: I see, thanks for the video

**Comment by deltanedas** [comment]
i think this is what broke actions

**Comment by ElectroJr** [comment]
Nah, it was my PR #16444
